bot/seasons/valentines/pickuplines.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class PickupLine:
+    """
+    A cog that gives random cheesy pickup lines.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def pickupline(self, ctx):
+        """
+        Gives you a random pickup line. Note that most of them are very cheesy!
+        """
+        with open(Path('bot', 'resources', 'valentines', 'pickup_lines.json'), 'r', encoding="utf8") as f:
```

Comment:
This doesn't need to be loaded every time the command is invoked. Have it loaded in the init or as a global constant.